### PR TITLE
Add identity provider picker to storybook

### PR DESCRIPTION
**Pull request by zanderz**
mobile storybook looking reasonable too

### CODE REVIEW

**Comment by MarcoPolo** [comment]
👀

File: shared/profile/user/proofs/index.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+const HoverBox = Styles.isMobile
+  ? Kb.Box
+  : Styles.styled(Kb.Box)({
+      ':hover': {backgroundColor: Styles.globalColors.blue4},
+    })
+
+class Providers extends React.Component<Props> {
+  render() {
+    return this.props.providers.map(a => (
```
[the remark this answers is not in the document]
Comment (zanderz) [reply shared/profile/user/proofs/index.js]: :thumbsup:

File: shared/profile/user/proofs/index.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+      <>
+        <Kb.Divider />
+        <HoverBox
+          fullWidth={true}
```
[the remark this answers is not in the document]
Comment (zanderz) [reply shared/profile/user/proofs/index.js]: :thumbsup:

File: shared/profile/user/proofs/index.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+              <Providers {...this.props} />
+            </Kb.ScrollView>
+          </Kb.Box2>
+          <HoverBox
+            centerChildren={true}
+            fullWidth={true}
+            onClick={this.props.onClickLearn}
+            style={styles.footer}
+          >
+            <Kb.Icon color={Styles.globalColors.black_50} fontSize={16} type="iconfont-info" />
```
[the remark this answers is not in the document]
Comment (zanderz) [reply shared/profile/user/proofs/index.js]: I think maybe it's better now with the other changes?

File: shared/profile/user/proofs/index.js [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+            <Kb.Text center={true} type="BodySmall" style={styles.footerText}>
```
[the remark this answers is not in the document]
Comment (zanderz) [reply shared/profile/user/proofs/index.js]: :thumbsup:

File: shared/profile/user/proofs/index.stories.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      desc: 'Mastodon instance',
+      icon: 'icon-placeholder-avatar-32',
+      name: 'hackers.town',
+      new: true,
+    },
+  ],
+}
+
+const load = () => {
+  Sb.storiesOf('Profile/Profile', module).add('Proof Providers List', () => <ProofsList {...props} />)
```
[the remark this answers is not in the document]
Comment (zanderz) [reply shared/profile/user/proofs/index.stories.js]: It scrolls for me; I just added a few more

File: shared/profile/user/proofs/index.js [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+    })
+
+class Providers extends React.Component<Props> {
+  render() {
+    return this.props.providers.map(a => (
+      <>
+        <Kb.Divider />
+        <HoverBox
+          fullWidth={true}
+          key={a.name}
```
[the remark this answers is not in the document]
Comment (zanderz) [reply shared/profile/user/proofs/index.js]: :white_check_mark:

**Comment by zanderz** [comment]
here is mobile - the input field doesn't look right:
image

File: shared/profile/user/proofs/index.js [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+              value={this.props.filter}
+            />
+          </Kb.Box>
+          <Kb.Box2 direction="vertical" fullWidth={true} style={styles.listContainer}>
+            <Kb.ScrollView>
+              <Providers {...this.props} />
+            </Kb.ScrollView>
+          </Kb.Box2>
+          <HoverBox
+            centerChildren={true}
```
[the remark this answers is not in the document]
Comment (zanderz) [reply shared/profile/user/proofs/index.js]: :white_check_mark:

File: shared/profile/user/proofs/index.js [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+              placeholderColor={Styles.globalColors.black_50}
+              multiline={false}
+              onChangeText={this.props.onSetFilter}
+              type="text"
+              style={styles.text}
+              value={this.props.filter}
+            />
+          </Kb.Box>
+          <Kb.Box2 direction="vertical" fullWidth={true} style={styles.listContainer}>
+            <Kb.ScrollView>
```
Comment (chrisnojima) [inline shared/profile/user/proofs/index.js]: just FYI we should not be using scrollviews like this. Use List2 instead

**Pull request merged**